js/hang-ui/src/Components/stats/handlers/audio.ts

```diff
@@ -46,13 +50,35 @@ export class AudioHandler extends BaseHandler {
 			codec?: string;
 		}>(this.props.audio.source.config);
 
-		const bitrate = this.peekSignal<number>(this.props.audio.source.bitrate);
+		const stats = this.peekSignal<AudioStats>(this.props.audio.source.stats);
 
 		if (!active || !config) {
 			this.context.setDisplayData("N/A");
 			return;
 		}
 
+		let bitrate: string | undefined;
+		if (stats && this.previousBytesReceived > 0) {
+			const bytesDelta = stats.bytesReceived - this.previousBytesReceived;
+			// Only calculate bitrate if there's actual data change
+			if (bytesDelta > 0) {
+				const bitsPerSecond = bytesDelta * 8 * (1000 / AudioHandler.POLLING_INTERVAL_MS);
```

Comment:
For best accuracy, also store `this.previousWhen = performance.now()` and subtract the actual time spent. Javascript won't fire intervals at exactly `AudioHandler.POLLING_INTERVAL_MS`, especially when the tab is backgrounded.